### PR TITLE
[readme] enhance readme 

### LINKED ISSUE #49: Update readme & documentation
## Issue Overview

Documentation is missing many points and readme lacks links. 

## Proposed Solutions or Ideas

Rewrite readme & documentation.

### PR DESCRIPTION
**Pull request by mrhouzlane**
# Update readme 

- enhance readme by adding future developments and a better design. 

## Types of change

- [ ] Feature
- [ ] Bug
- [X] Enhancement

## Comments (optional)

Closes #49

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +53,7 @@ NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=<your_wallet_connect_project_id_here>
 ```
```
Comment (edulanasca) [inline README.md:53]: I think this backlashes should go in line 41 after `yarn install`. 
This is causing line 42 `### Managing... ` look like that is part of the commands.

**Review by edulanasca — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mrhouzlane) [reply README.md:53]: it's already after yarn install

Comment (mrhouzlane) [reply README.md:53]: Those are env variables that you add ..

**Comment by edulanasca** [comment]
@mrhouzlane pls check it

**Comment by mrhouzlane** [comment]
You can merge & squash this.

**Review by edulanasca — APPROVED** [review]

**Pull request merged**